editor/src/clj/editor/tile_map.clj

```diff
@@ -586,11 +586,11 @@
         (make-layer-node self tile-layer)))))
 
 (g/defnk produce-scene
-  [_node-id child-scenes]
-  {:node-id  _node-id
-   :aabb geom/null-aabb
-   :renderable {:passes [pass/selection]}
-   :children child-scenes})
+  [_node-id child-scenes scene-aabb]
+  {:node-id           _node-id
+   :aabb              scene-aabb
+   :renderable        {:passes [pass/selection]}
+   :children          child-scenes})
```

Comment:
the next `produce-node-outline` the same code formatting when create data, so it makes sense to have it here as well